tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -29,19 +29,24 @@ def pull_project_peds(session_id: str, projects: set):
             )
             for project_guid in projects:
                 project_guid = project_guid.rstrip()
-                r = s.get(
-                    f"{SEQR_URL}/api/project/{project_guid}/details"
+                family_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/get_families"
                 )
-
-                if r.status_code != requests.codes["ok"]:
+                individual_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/get_individuals"
+                )
+                if family_r.status_code != requests.codes["ok"]:
+                    logger.info(f"Could not find {project_guid} in seqr")
+                    errors.write(f"{project_guid}\n")
+                elif individual_r.status_code != requests.codes["ok"]:
```

Comment:
when would `family_r` be fine and `individual_r` have an error?

Reply:
Hm, I don't know if there's a scenario where one is fine, but the other isn't. We should probably fail if either of these response codes have an error - does that sound good?

Reply:
that works for me!